src/amo/reducers/collections.js

```diff
@@ -573,7 +573,7 @@ type DeleteCollectionBySlugAction = {|
   payload: {| slug: string |},
 |};
 
-export const deleteCollectionBySlug = (
+export const unloadCollectionBySlug = (
   slug: string
 ): DeleteCollectionBySlugAction => {
```

Comment:
We also need to rename `DeleteCollectionBySlugAction` to `UnloadCollectionBySlugAction`, and the const `DELETE_COLLECTION_BY_SLUG` should be renamed to `UNLOAD_COLLECTION_BY_SLUG`.